eventlivedata/live_db.go

```diff
@@ -0,0 +1,65 @@
+package eventlivedata
+
+import (
+	"fmt"
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/dynamodb"
+	"os"
+	"strconv"
+)
+
+type live_db_adapter interface {
```

Comment:
Can we make this camel case rather than snake case to avoid confusing with package naming?

Reply:
fixed in e289132